specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-06-01-preview/apimdeployment.json

```diff
@@ -589,103 +596,6 @@
         },
         "x-ms-long-running-operation": true
       }
-    },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/updatecertificate": {
```

Comment:
This API appears to still be in active use. While this is a preview API, has your deprecation policy been forgiving enough? A new API version would be a cleaner way to drop these now.